otwcode/otwarchive
# AO3-5332 Remove creator from work when they remove themselves from last chapter they created

## Issue

https://otwarchive.atlassian.net/browse/AO3-5332

## Purpose

When a co-creator removes themselves from the last chapter for which they were credited, it should also remove them as a co-creator of the work and remove all associated edit permissions etc.

## Testing

- Add at least two chapters as co-creator. 
- Using chapter edit and/or manage, remove yourself from one of the chapters - verify still have edit access to work.
- Remove yourself from from other (last credited) chapter - verify redirected to user screen, verify no longer have edit access to work.
